### PR TITLE
Create 2023-KCD-Munich

### PR DESCRIPTION
**Pull request by karenhchu**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [ ] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [ ] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [ ] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [ ] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [x] Headings are using Title Case
- [ ] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [ ] Have tested with `npm run test` and resolved all errors
- [ ] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/events/2023-KCD-Munich [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+description = "Fermyon engineers Danielle Lancashire and Ryan Levick, along with Tom Gehrke from Liquid Reply, will be giving a \"Hands on with WebAssembly Microservices and Kubernetes\" tutorial at Kubernetes Community Day (KCD) Munich. Don't miss them on Monday July 17 @ 4 PM as they help introduce attendees to WebAssembly."
+
+[extra]
+type = "event"
+eventdates = "July 17 - 18 2023"
+eventexpires = "2023-07-19T01:01:01Z"
+url = "https://www.kcdmunich.de/"
+image = "static/image/avatar.png"
+
+---
```
Comment (karthik2804) [inline content/events/2023-KCD-Munich:12]:
```suggestion
---

```

**Review by karthik2804 — CHANGES_REQUESTED** [review]

File: content/events/2023-KCD-Munich [hunk 2]
```diff
@@ -0,0 +1,12 @@
+title = "Kubernetes Community Day Munich"
+date = "2023-07-07T00:00:01Z"
+description = "Fermyon engineers Danielle Lancashire and Ryan Levick, along with Tom Gehrke from Liquid Reply, will be giving a \"Hands on with WebAssembly Microservices and Kubernetes\" tutorial at Kubernetes Community Day (KCD) Munich. Don't miss them on Monday July 17 @ 4 PM as they help introduce attendees to WebAssembly."
```
Comment (endocrimes) [inline content/events/2023-KCD-Munich:3]: Probably wanna include this as Munich time rather than EST 😅

Comment (karenhchu) [reply content/events/2023-KCD-Munich:3]: I'm not seeing where it says EST? It should be Z(Zulu)/UTC. This line is also for the publishing time for when this file goes live.

Comment (karthik2804) [reply content/events/2023-KCD-Munich:3]: Possibly the time in the description?

**Review by karthik2804 — APPROVED** [review]

File: content/events/2023-KCD-Munich (hunk 2 above)
Comment (karenhchu) [reply content/events/2023-KCD-Munich:3]: Oh I was hoping it was implied it'd be local time relative to where the conf is. Should I add CEST?

File: content/events/2023-KCD-Munich [hunk 3]
```diff
@@ -0,0 +1,13 @@
+title = "Kubernetes Community Day Munich"
+date = "2023-07-07T00:00:01Z"
+description = "Fermyon engineers Danielle Lancashire and Ryan Levick, along with Tom Gehrke from Liquid Reply, will be giving a \"Hands on with WebAssembly Microservices and Kubernetes\" tutorial at Kubernetes Community Day (KCD) Munich. Don't miss them on Monday July 17 @ 4 PM CEST as they help introduce attendees to WebAssembly."
```
Comment (endocrimes) [inline content/events/2023-KCD-Munich:3]:
```suggestion
description = "Fermyon engineers Danielle Lancashire and Ryan Levick, along with Tom Gehrke from Liquid Reply, will be giving a \"Hands on with WebAssembly Microservices and Kubernetes\" tutorial at Kubernetes Community Day (KCD) Munich. Don't miss them on Monday July 17 @ 10:40 AM CEST as they help introduce attendees to WebAssembly."

As much as i wish it was 4pm, sadly it is not :(
```

Comment (itowlson) [reply content/events/2023-KCD-Munich:3]: @endocrimes that last line isn't meant to be part of the suggestion patch right?!

Comment (endocrimes) [reply content/events/2023-KCD-Munich:3]: it was not 😂

Comment (endocrimes) [reply content/events/2023-KCD-Munich:3]: fixed!

**Comment by tpmccallum** [comment]
This DCO is now passing on the last 2 commits for some reason.
I just went and created a new PR to get this up asap.
https://github.com/fermyon/developer/pull/728
Closing this one if favour of the other.
Thanks for adding this event in the first place! Much appreciated.

**Pull request closed without merging**